cache_system/src/backend/policy/mod.rs

```diff
@@ -418,23 +441,32 @@ pub trait Subscriber: Debug + Send + 'static {
     type V: Clone + Debug + Send + 'static;
 
     /// Get value for given key if it exists.
-    fn get(&mut self, _k: &Self::K) -> Vec<ChangeRequest<'static, Self::K, Self::V>> {
+    fn get(&mut self, _k: &Self::K, _now: Time) -> Vec<ChangeRequest<'static, Self::K, Self::V>> {
         // do nothing by default
         vec![]
     }
 
     /// Set value for given key.
     ///
     /// It is OK to set and override a key that already exists.
```

Comment:
I think it might be worth adding a comment here with the rationale for including `Time` as a parameter as it my not be obvious at some later date:

```suggestion
    /// It is OK to set and override a key that already exists.
    ///
    /// The current time `now` is provided as a parameter so that all policies and 
    /// backends use a unified timestamp rather than their own provider, which is 
    /// more consistent and performant.
```